src/edit/items/ItemForm.js

```diff
@@ -266,7 +266,7 @@ class ItemForm extends React.Component {
 
     const labelLocation = get(holdingLocation, ['name'], '');
     const labelCallNumber = holdingsRecord.callNumber || '';
-    const effectiveLocation = locationsById[initialValues.effectiveLocation.id];
+    const effectiveLocation = locationsById[initialValues?.effectiveLocation?.id];
```

Comment:
The effectiveLocation null check is what matters here.  The initialValues check is probably not necessarily since it would have thrown an error earlier on line 216.  But I don't think it hurts.